docs/sphinx_project/conf.py

```diff
@@ -16,9 +16,9 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath("../.."))
+import pygromos  # noqa: F401
 
-import pygromos
+sys.path.insert(0, os.path.abspath("../.."))
```

Comment:
uh here the order changed, why? maybe the sys path can be removed anyways, as this should not be an issue in an proper env.